src/termio/shell_integration.zig

```diff
@@ -203,8 +203,6 @@ test "setup features" {
 /// our script's responsibility (along with disabling POSIX
 /// mode).
 ///
-/// This approach requires bash version 4 or later.
```

Comment:
This was never quite true. The problem was with Apple's bash version of bash 3.2 (#2966).